vignettes/customizing-dccvalidator.Rmd

```diff
@@ -30,7 +30,7 @@ To create a custom version of the app, you'll need to follow these steps:
 1. Fork the [dccvalidator](https://github.com/Sage-Bionetworks/dccvalidator/blob/master/config.yml)
    GitHub repository.
 1. Create a new configuration in the `config.yml` file. Note that any values you
-   do not customize will be inherited from the default configuration.
+   do not customize will be inherited from the default configuration. YML format requires a default configuration.
```

Comment:
Can we tweak the wording of this slightly? It's the config R package that requires the default configuration (not anything to do with the yaml format) but that's probably more detail than is needed. Since the config.yml file already has the default section, maybe we can say something like "The configuration file must have a `default` configuration."